### PR TITLE
removed None

### PR DESCRIPTION
**Pull request by jrmlhermitte**
@danielballan small change

### CODE REVIEW

File: bluesky/callbacks/best_effort.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def start(self, doc):
 
         # Prepare a guess about the dimensions (independent variables) in case
         # we need it.
-        motors = self._start_doc.get('motors') or None
+        motors = self._start_doc.get('motors')
```
Comment (tacaswell) [inline bluesky/callbacks/best_effort.py]: This changes behavior in the case when the `get` returns an empty list, not sure of the consequence of that though....

Comment (jrmlhermitte) [reply bluesky/callbacks/best_effort.py]: hmm that's interesting. I'm more in favor of a consistent result that uncertain. Tests pass but I think this PR should be closed. What do yo think?

**Comment by tacaswell** [comment]
I _think_ leaving the `or None` ensures that we don't get empty lists going through (and hence always have a 'decent' guess), but I leave to @danielballan.

**Comment by danielballan** [comment]
I think if a plan hints `motors=[]`, that could mean something different than omitting `motors` from the hint. I think this PR should probably be merged.

**Pull request merged**